Storage/src/Connection/Rest.php

```diff
@@ -304,6 +322,10 @@ public function insertObject(array $args = [])
             ]
         ];
 
+        if (isset($args['ifGenerationMatch'])) {
```

Comment:
Can we try and merge this `if(isset())` and the one above?

Reply:
Yes, done in this commit [e8174b5](https://github.com/googleapis/google-cloud-php/pull/5730/commits/e8174b581f2af6ee61c7822010d426e46fc92a80)

Thanks for pointing out.